web_widget_image_webcam/static/src/xml/web_widget_image_webcam.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2016 Siddharth Bhalgami <siddharth.bhalgami@techreceptives.com>
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). -->
+<template>
+    <t t-extend="FieldBinaryImage">
+        <t t-jquery=".oe_form_field_image_controls" t-operation="after">
```

Comment:
Problem is that [there are other modules that add icons there](https://github.com/OCA/web/tree/9.0/web_widget_image_download). If each module adds a new bar just because you don't want to overload the original one, then we'd end up with lots of bars... This could be fixed by an addon that just gets that bar and puts it in some other layout that makes it fit better when there are lots of icons, but I think I prefer to use the same icons bar that is already present instead of creating new ones.